### PR TITLE
[MB-1594] Add menu item to configure Urban Airship credentials

### PR DESCRIPTION
**Pull request by rlepinski**
Adds a menu to configure UA.

### CODE REVIEW

File: Assets/UrbanAirship/Editor/UAAssetPostProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+		{
+			if (importedAssets.Any(s => s.Contains("urbanairship") || s.ToLower().Contains("airshipconfig")))
+			{
+				UAUtils.UpdateManifests();
+				UnityEngine.Debug.Log ("Updated Urban Airship Manifests.");
+				AssetDatabase.Refresh();
+			}
+
+			if(deletedAssets.Any(s => s.ToLower().Contains("airshipconfig")))
```
Comment (hcrowell) [inline Assets/UrbanAirship/Editor/UAAssetPostProcessor.cs]: if`space`(

File: Assets/UrbanAirship/Editor/UAConfig.cs [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+			rootDict.SetBoolean("inProduction", InProduction);
+
+
+			File.WriteAllText(plistPath, plist.WriteToString());
+		}
+
+		private void GenerateAndroidAirshipConfig()
+		{
+			string assetsDirecotry = Path.Combine(Application.dataPath, "Plugins/Android/assets");
```
Comment (hcrowell) [inline Assets/UrbanAirship/Editor/UAConfig.cs]: assetsDirectory*

File: Assets/UrbanAirship/Editor/UAAssetPostProcessor.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+using System;
+using UnityEditor;
+using System.Linq;
+using UnityEngine;
+using System.IO;
+
+namespace UrbanAirship
+{
+	[InitializeOnLoad]
+	public class UAAssetPostprocessor : AssetPostprocessor
```
Comment (hcrowell) [inline Assets/UrbanAirship/Editor/UAAssetPostProcessor.cs]: nitpick: match file name case sensitivity? ie. UAAssetPostProcessor ?

File: Assets/UrbanAirship/Editor/UAPostBuild.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+using UnityEditor.iOS.Xcode;
+
+namespace UrbanAirship
+{
+
+	public class UAPostBuild
+	{
+
+		[PostProcessBuildAttribute(1)]
+		public static void OnPostprocessBuild(BuildTarget target, string buildPath)
```
Comment (hcrowell) [inline Assets/UrbanAirship/Editor/UAPostBuild.cs]: nitpick: could we match either capital `P` (PostProcessBuildAttribute, OnPostProcessBuild) or little `p` (PostprocessBuildAttribute, OnPostprocessBuild) ?

File: Assets/UrbanAirship/Editor/UAConfigEditor.cs [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+			config = UAConfig.LoadConfig();
+		}
+
+		void OnGUI()
+		{
+			GUILayout.Label ("Production App Credentials", EditorStyles.boldLabel);
+			config.ProductionAppKey = EditorGUILayout.TextField("\t App Key", config.ProductionAppKey);
+			config.ProductionAppSecret = EditorGUILayout.TextField("\t App Secret", config.ProductionAppSecret);
+
+			GUILayout.Label ("Development Credentails", EditorStyles.boldLabel);
```
Comment (hcrowell) [inline Assets/UrbanAirship/Editor/UAConfigEditor.cs]: Development App* Credentials*

**Comment by hcrowell** [comment]
:tangerine:

**Comment by marc-scig** [comment]
:recycle:

**Pull request merged**